gapis/api/vulkan/resources.go

```diff
@@ -1115,8 +1117,12 @@ func commonShaderDataGroups(ctx context.Context,
 							currentSetData = append(currentSetData, api.CreateLinkedDataValue("url", samplerPath, api.CreatePoDDataValue("VkSampler", samplerHandle)))
 
 							viewHandle := descInfo.ImageView()
-							viewPath := path.NewField("ImageViews", resolve.APIStateAfter(path.FindCommand(cmd), ID)).MapIndex(viewHandle)
-							currentSetData = append(currentSetData, api.CreateLinkedDataValue("url", viewPath, api.CreatePoDDataValue("VkImageView", viewHandle)))
+							//viewPath := path.NewField("ImageViews", resolve.APIStateAfter(path.FindCommand(cmd), ID)).MapIndex(viewHandle)
+
+							imageView, _ := vkState.ImageViews().Lookup(viewHandle)
+							imageHandle := imageView.Image().VulkanHandle()
+							imageViewPath := path.NewImageView(fmt.Sprintf("Image<%d>", imageHandle))
```

Comment:
This is the hacky part I want to discuss

Reply:
We used to return a state path. (CmdID->ImageViews->Handle). 
I think we should create something similar to "Follow" (FollowResource) which takes that existing pah (CmdID->ImageViews->Handle) and returns a new Resource Path. This resource could contain the type (Texture in this case) and the information the UI would need for it. (Handle I believe) From that ResourcePath, we could determine which windows need to be in front (texture view in this case) and go to the correct object.

This lets us do this from anywhere in the UI, instead of only being able to do it from the TextureView. (For example, if someone clicked on an ImageView in the state-view we could also do this, or a pipeline->pipeline View, or buffer -> Model view etc etc).